core/crypto/sr25519_types.hpp

```diff
@@ -33,24 +33,32 @@ namespace kagome::crypto {
        */
       enum {
         PROOF_SIZE = SR25519_VRF_PROOF_SIZE,
-        OUTPUT_SIZE = SR25519_VRF_OUTPUT_SIZE
+        OUTPUT_SIZE = SR25519_VRF_OUTPUT_SIZE,
+        RAW_OUTPUT_SIZE = SR25519_VRF_RAW_OUTPUT_SIZE
       };
     }  // namespace vrf
 
   }  // namespace constants::sr25519
 
   using VRFValue = boost::multiprecision::uint256_t;
+  using VRFRawOutput = boost::multiprecision::uint128_t;
 
   using VRFProof = std::array<uint8_t, constants::sr25519::vrf::PROOF_SIZE>;
 
   struct VRFOutput {
-    VRFValue value;
+    VRFValue output;
+    VRFRawOutput raw_output;
     VRFProof proof{};
```

Comment:
Please keep only the fields that we use for serialization. Also keep in mind that serialization should be compatible with Substrate, where output has size 32 and proof 64. https://github.com/paritytech/substrate/blob/7eb937e8ff7b7de2cd77a4116677e5e20605e25c/core/consensus/babe/primitives/src/digest.rs#L111-L113